### PR TITLE
update pins

### PR DESCRIPTION
**Pull request by ocefpaf**

### CODE REVIEW

**Comment by ocefpaf** [comment]
Travis-CI is passing on my instance, so merging this.

https://travis-ci.org/ocefpaf/python-feedstock/builds/312990050

File: recipe/meta.yaml [hunk 1]
```diff
@@ -39,22 +39,22 @@ requirements:
...
-    - ncurses 5.9*      # [linux]
+    - ncurses 5.9      # [linux]
   run:
     - openssl 1.0.*     # [unix]
-    - readline 6.2*     # [unix]
-    - sqlite 3.13.*     # [unix]
-    - tk 8.5.*          # [unix]
+    - readline 7.0     # [unix]
+    - sqlite 3.20.*     # [unix]
+    - tk 8.6.*          # [unix]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Even though this was bumped to 8.6, it is still linking against 8.5. Also it appears to now being using the system Tcl/TK, which it [was not doing before]( https://travis-ci.org/conda-forge/python-feedstock/builds/297158425#L7201-L7203 ).

ref: https://github.com/conda-forge/python-feedstock/blob/c2fe3e8753fa50e30b4a27c7735dfd6c0a1dd08a/recipe/build.sh#L35
ref: https://travis-ci.org/ocefpaf/python-feedstock/builds/312990050#L7221-L7222

Comment (jakirkham) [reply recipe/meta.yaml]: Try to address this in PR ( https://github.com/conda-forge/python-feedstock/pull/162 ).

File: recipe/meta.yaml [hunk 2]
```diff
@@ -39,22 +39,22 @@ requirements:
...
     - 7za               # [win]
     - bzip2 1.0.*       # [unix]
-    - curl              # [win]
+    - curl >=7.44.0,<8              # [win]
     - openssl 1.0.*     # [unix]
-    - readline 6.2*     # [unix]
-    - sqlite 3.13.*     # [unix]
-    - tk 8.5.*          # [unix]
+    - readline 7.0     # [unix]
+    - sqlite 3.20.*     # [unix]
```
Comment (jakirkham) [inline recipe/meta.yaml]: So should [`sqlite` in the pinnings]( https://github.com/conda-forge/conda-forge.github.io/blob/24a34313f341c352ea87806b2dc63de672ff881a/scripts/pin_the_slow_way.py#L96 ) be updated as it points to `3.13.*` currently.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -39,22 +39,22 @@ requirements:
...
+    - tk 8.6.*          # [unix]
```
Comment (jakirkham) [inline recipe/meta.yaml]: So should [`tk` in the pinnings]( https://github.com/conda-forge/conda-forge.github.io/blob/24a34313f341c352ea87806b2dc63de672ff881a/scripts/pin_the_slow_way.py#L98 ) be updated as it points to `8.5.*` currently.

**Pull request merged**